content/pt-br/docs/tasks/administer-cluster/limit-storage-consumption.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Limitar o consumo de armazenamento
+content_type: task
+weight: 240
+---
+
+<!-- overview -->
+
+Este exemplo demonstra como limitar a quantidade de armazenamento consumido em um namespace.
+
+Os seguintes recursos são usados na demonstração: [`ResourceQuota`](/pt-br/docs/concepts/policy/resource-quotas/), [`LimitRange`](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/), e [`PersistentVolumeClaim`](/pt-br/docs/concepts/storage/persistent-volumes/).
+
+## {{% heading "prerequisites" %}}
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+## Cenário: Limitando o consumo de armazenamento.
+
+O cluster-admin está operando um cluster em nome de uma população de usuários e o admin quer controlar quanto armazenamento um único namespace pode consumir para controlar custos.
+
+O admin gostaria de limitar:
```

Comment:
Suggestion: spell out "administrador" and "administrador do cluster" in these sentences, instead of keeping the original English version.